okr/2023-s2.md

```diff
@@ -127,6 +127,7 @@
 ### Objectives:
 
 - Make Force Bridge stable and faster;
+- Optimize Force Bridge early warning;
```

Comment:
What is early warning

Reply:
> What is early warning

I updated the description to be a little more specific